examples/create-react-app/src/components/Posts.js

```diff
@@ -14,7 +14,13 @@ export const allPostsQuery = `
 `
 
 export default function Posts() {
-  const { loading, data, error, refetch } = useQuery(allPostsQuery)
+  const { loading, data, error, refetch } = useQuery(allPostsQuery, {
+    refetchAfterMutations: [
+      {
+        mutation: createPostMutation
```

Comment:
Maybe I missed something, but I thought that the mutation we're specifying here is the **name** of the mutation, not the string of the mutation itself. Is it intentional that we are specifying the full body of the mutation instead of its name?

Asking especially because I can surely see the advantage in doing so (you don't have to hardcode names, since they in the application they have little relevance), but on the other hand I wonder if using as key the whole body of a mutation is a robust approach.

Reply:
Yes, it's intentionally the entire mutation string.
I thought of that because it's what we already have handy for matching with a mutation and avoiding mistakes.
Other ideas are welcome :)

Reply:
sounds good, let's make sure this is clear in the docs because it's not obvious